### ISSUE TITLE
Support toPkcs when issue and renew

### ISSUE THREAD

**Issue by sjtuross** [acmesh-official/acme.sh#3893]
1. Generating pfx/p12 file when issue and renew succeeds (probably by adding toPkcs parameter)
2. Add DEPLOY_SSH_PFXFILE environment variable to reference the pfx/p12 file in SSH deploy hook

With the above, it can make local and remote deployment easier for some applications like UniFi Controller, Plex, Emby, Jellyfin etc.

**Comment by maffe** [comment]
Duplicate of #3218?

**Comment by Baenwort** [comment]
I don't think it is a duplicate as the #3218 is seeking how to do it via script and this one is seeking a GUI/built in way to accomplish a .pfx generation.

I'd say this would be handy and helpful for me at least.
